WordPress/src/main/java/org/wordpress/android/ui/stats/StatsFollowersFragment.java

```diff
@@ -38,7 +38,7 @@
 public class StatsFollowersFragment extends StatsAbstractListFragment {
     public static final String TAG = StatsFollowersFragment.class.getSimpleName();
 
-    private HashSet<String> dotComUserBlogsURL = new HashSet<>();
+    private Map<String, String> userBlogs = new HashMap<>();
```

Comment:
Can userBlogs simply be `Map<String, Integer>` so blogID doesn't need to be converted to/from a string?